src/plugins/discord/main.ts

```
@@ -49,6 +49,20 @@ export const backend = createBackend<
       ctx.ipc.send('peard:setup-time-changed-listener');
     });
 
+    ctx.ipc.on(
+      'discord:youtube-info',
+      (info: unknown) => {
```

ESLint

[error] Replace `⏎······'discord:youtube-info',⏎·····` with `'discord:youtube-info',`

(prettier/prettier)